hudi-client/src/main/java/org/apache/hudi/HoodieWriteClient.java

```diff
@@ -586,6 +586,7 @@ public boolean savepoint(String commitTime, String user, String comment) {
           HoodieTimeline.compareTimestamps(commitTime, lastCommitRetained, HoodieTimeline.GREATER_OR_EQUAL),
           "Could not savepoint commit " + commitTime + " as this is beyond the lookup window " + lastCommitRetained);
 
+      jsc.setJobGroup(this.getClass().getSimpleName(), "Collecting latest files in partition");
```

Comment:
In general lets provide some context into what higher level context, the action is being performed i.e savepoints, compaction, rollbacks. etc . In that spirit, change to `Collecting latest files for savepoint` ? 

Also wonder if we can include the `commitTime` in the detail i.e `Collecting latest files for savepoint 20200205010000`. This way, you can just go to past runs on spark history server and relate them to commits on hudi.. Even better, if someone is running deltastreamer in continuous mode, then they can see activity for commits over time